r/R/dplyr.R

```diff
@@ -390,6 +409,25 @@ collect.arrow_dplyr_query <- function(x, as_data_frame = TRUE, ...) {
         tab <- RecordBatch$create(!!!cols)
       }
     }
+    # Arrange rows
+    # TODO: support sorting by expressions, not just field names
```

Comment:
So you can sort on an expression, but that expression won't just `mutate()`? Hmm.

Reply:
Something like that, yes. But it might be cleaner to construct the Vector of mutated values to sort on outside of the Table/RecordBatch so we don't need to deal with column naming and removing it after. I'm still experimenting with that.

Reply:
Done in 0ba20fb57c542e3099b047c792579819685c8888